src/Equinox.MessageDb/Tracing.md

```diff
@@ -0,0 +1,43 @@
+All tags are prefixed with `eqx.`
```

Comment:
@bartelink gathered the tags here. I made some changes as I was writing them out that I haven't actually put into practise. LMK if you find something missing or unclear

Reply:
Oh yeah, doh. In that case, a one liner pointing there at the top of each tracing.fs please
Hm on reflection maybe the answer to all this is to have a `module Tags` inside each `module Tracing` - the xmldoc can go on the literal and you dont need to remember to update an MD file that has no teeth (MD can still call out key ones)

Reply:
MD file is just for the purposes of this PR (so you can comment on individual lines and we have it in a central place while ironing this out). Fully aligned with moving the docs into code once it's ironed